Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -140,6 +143,21 @@ bool JITData::tryInitialize(VM& vm, CodeBlock* codeBlock, const JITCode& jitCode
             success &= attemptToWatch(codeBlock, m_globalObject->structureCacheClearedWatchpointSet(), watchpoint);
             break;
         }
+        case LinkerIR::Type::StringValueOfWatchpointSet: {
+            auto& watchpoint = m_watchpoints[indexOfWatchpoints++];
+            success &= attemptToWatch(codeBlock, m_globalObject->stringValueOfWatchpointSet(), watchpoint);
+            break;
+        }
+        case LinkerIR::Type::StringToStringWatchpointSet: {
+            auto& watchpoint = m_watchpoints[indexOfWatchpoints++];
+            success &= attemptToWatch(codeBlock, m_globalObject->stringToStringWatchpointSet(), watchpoint);
+            break;
+        }
+        case LinkerIR::Type::StringSymbolToPrimitiveWatchpointSet: {
+            auto& watchpoint = m_watchpoints[indexOfWatchpoints++];
+            success &= attemptToWatch(codeBlock, m_globalObject->stringSymbolToPrimitiveWatchpointSet(), watchpoint);
+            break;
+        }
```

Comment:
Let's place these case statement in the order they are declared in DFGJITCode.h below.  That will give the C++ compiler an easier job of optimizing this switch statement.  Plus, it'll be easier for a human to read the code if the order is the same in both lists.